### PR TITLE
Fixdocs

### PR DESCRIPTION
**Pull request by dnadeau4**

### CODE REVIEW

File: cdat_utilities/Masking_Data_Land-Sea_masks/Masking_Data_Land-Sea_masks.html [hunk 1]
```diff
@@ -11766,73 +13108,165 @@
...
-</div>
-<div class="inner_cell">
+</div><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
-<h1 id="Masking-Data-Land-Sea-Masks-">Masking Data Land-Sea Masks<a class="tocSkip" id="top"> </a><a class="anchor-link" href="#Masking-Data-Land-Sea-Masks-">&#182;</a></h1><p>In this example we will show how to generate masks, including land/sea masks</p>
+<h1 id="Masking-Data-Land-Sea-Masks-">Masking Data Land-Sea Masks<a class="tocSkip" id="top"> </a><a class="anchor-link" href="file:///home/nadeau1/Downloads/Masking_Data_Land-Sea_masks.html#Masking-Data-Land-Sea-Masks-">¶</a></h1><p>In this example we will show how to generate masks, including land/sea masks</p>
 <p>© The CDAT software was developed by LLNL. This tutorial was written by Charles Doutriaux. This work was performed under the auspices of the U.S. Department of Energy by Lawrence Livermore National Laboratory under Contract DE-AC52-07NA27344.</p>
-<p><a href="Masking_Data_Land-Sea_masks.ipynb">Download the Jupyter Notebook</a></p>
+<p><a href="file:///home/nadeau1/Downloads/Masking_Data_Land-Sea_masks.ipynb">Download the Jupyter Notebook</a></p>
```
Comment (davis278) [inline cdat_utilities/Masking_Data_Land-Sea_masks/Masking_Data_Land-Sea_masks.html]: The notebook and all links within the html version of the notebook need to remain as relative links so the href="file:///home/nadeau1/Downloads/Masking_Data_Land-Sea_masks.ipynb" needs to remain href="Masking_Data_Land-Sea_masks.ipynb". All of the places in this file where "file:///home/nadeau1/Downloads/" exists needs to be removed.

File: vcs/Animations/Animations.html [hunk 2]
```diff
@@ -11743,13 +13085,13 @@
 </style>
 
 <!-- Custom stylesheet, it must be in the same directory as the html file -->
-<link rel="stylesheet" href="custom.css">
+<link rel="stylesheet" href="file:///home/nadeau1/Downloads/custom.css">
```
Comment (davis278) [inline vcs/Animations/Animations.html]: Using a local file will break the html page when posted to the web.

File: vcs/Colormap_Create/Color_map_create_own.html [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
-<html>
-<head><meta charset="utf-8" />
-<title>Color_map_create_own</title><script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.1.10/require.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+<!-- saved from url=(0056)file:///home/nadeau1/Downloads/Color_map_create_own.html -->
+<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+
+<title>Color_map_create_own</title>
+
+<script src="./Color_map_create_own_files/require.min.js"></script>
```
Comment (davis278) [inline vcs/Colormap_Create/Color_map_create_own.html]: Not sure it is a good idea to change away from the direct links for the require.min.js and jquery.min.js files - is this an artifact of having the html file saved from the local copy in the Downloads folder?

File: vcs/Colormap_Create/Color_map_create_own.html [hunk 4]
```diff
@@ -11766,37 +13108,129 @@
...
   <div tabindex="-1" id="notebook" class="border-box-sizing">
     <div class="container" id="notebook-container">
 
 <div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
-</div>
-<div class="inner_cell">
+</div><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
-<h1 id="Create-Your-Own-Colormap-">Create Your Own Colormap<a class="tocSkip" id="top"> </a><a class="anchor-link" href="#Create-Your-Own-Colormap-">&#182;</a></h1><p>Detailed reference document of the VCS colormap can be found at <a href="http://localhost:8888/view/vcs/Colormap_Create/Color_map_create_own.png">here</a>. Colormap for example 4 provided by Karl Taylor.</p>
+<h1 id="Create-Your-Own-Colormap-">Create Your Own Colormap<a class="tocSkip" id="top"> </a><a class="anchor-link" href="file:///home/nadeau1/Downloads/Color_map_create_own.html#Create-Your-Own-Colormap-">¶</a></h1><p>Detailed reference document of the VCS colormap can be found at <a href="http://localhost:8888/view/vcs/Colormap_Create/Color_map_create_own.png">here</a>. Colormap for example 4 provided by Karl Taylor.</p>
```
Comment (davis278) [inline vcs/Colormap_Create/Color_map_create_own.html]: Same local file link problem as in other html files.

File: scientific/Portrait_Plot/ReusablePortraitPlot.html [hunk 5]
```diff
@@ -11743,13 +13085,13 @@
 </style>
 
 <!-- Custom stylesheet, it must be in the same directory as the html file -->
-<link rel="stylesheet" href="custom.css">
+<link rel="stylesheet" href="file:///home/nadeau1/Downloads/custom.css">
```
Comment (davis278) [inline scientific/Portrait_Plot/ReusablePortraitPlot.html]: In addition to using a local copy of the custom.css file, it looks like this version of the custom.css file is noticeably different from the old version. (This phenomena shows up in many if not all the html files I've looked at so far.)

**Comment by dnadeau4** [comment]
@davis278 you can change all the hardlink.  Not sure why it did this when I saved from my notebooks.

**Pull request merged**